.github/workflows/acceptance-tests.yml

```diff
@@ -97,7 +97,6 @@ jobs:
       mongodb_atlas_base_url: ${{ inputs.atlas_cloud_env == 'qa' && vars.MONGODB_ATLAS_BASE_URL_QA || vars.MONGODB_ATLAS_BASE_URL }}
       mongodb_atlas_project_owner_id: ${{ inputs.atlas_cloud_env == 'qa' && vars.MONGODB_ATLAS_PROJECT_OWNER_ID_QA || vars.MONGODB_ATLAS_PROJECT_OWNER_ID }}
       mongodb_atlas_teams_ids: ${{ inputs.atlas_cloud_env == 'qa' && vars.MONGODB_ATLAS_TEAMS_IDS_QA || vars.MONGODB_ATLAS_TEAMS_IDS }}
-      mongodb_atlas_username: ${{ inputs.atlas_cloud_env == 'qa' && vars.MONGODB_ATLAS_USERNAME_CLOUD_QA || vars.MONGODB_ATLAS_USERNAME_CLOUD_DEV }}
```

Comment:
noticed there is not need to have different values for dev/qa, unified to single GH variable MONGODB_ATLAS_USERNAME. Checked [tests are passing in qa](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/11255205368) as well.
Once pr is merged will remove MONGODB_ATLAS_USERNAME_CLOUD_DEV and MONGODB_ATLAS_USERNAME_CLOUD_QA